paasta_tools/instance/kubernetes.py

```diff
@@ -641,7 +641,7 @@ async def kubernetes_status_v2(
                 kube_client, job_config, job_config.get_kubernetes_namespace()
             )
         )
-        tasks.append(autoscaling_task)
+        tasks.append(autoscaling_task)  # type: ignore  # PAASTA-18698; ignoring due to type mismatch
```

Comment:
```suggestion
        tasks.append(autoscaling_task)  # type: ignore  # PAASTA-18698; ignoring due to unexpected type mismatch
```
might be a little more descriptive, but this is fine too